index.bs

```diff
@@ -171,32 +172,19 @@ Resource names starting with U+002D HYPHEN-MINUS (-) are reserved; requesting th
 ## Lock Managers ## {#lock-managers}
 <!-- ====================================================================== -->
 
-A [=/user agent=] has a <dfn>lock manager</dfn> for each [=/origin=], which encapsulates the state of all [=lock-concept|locks=] and [=lock requests=] for that origin.
+A [=/user agent=] has a <dfn>lock manager</dfn> for each [=/storage bucket=], which encapsulates the state of all [=lock-concept|locks=] and [=lock requests=] associated with a [=/storage bucket=].
 
-Note: Pages and workers ([=/agents=]) on a single [=/origin=] opened in the same user agent share a lock manager even if they are in unrelated [=/browsing contexts=].
+Note: Pages and workers ([=/agents=]) sharing a [=/storage bucket=] opened in the same user agent share a [=/lock manager=] even if they are in unrelated [=/browsing contexts=].
 
 <div algorithm>
 To <dfn>obtain a lock manager</dfn>, given an [=/environment settings object=] |environment|, run these steps:
 
-1. Let |origin| be |environment|'s [=/origin=].
-1. If |origin| is an [=/opaque origin=], then return failure.
-1. Return |origin|'s associated [=/lock manager=].
+1. Let |bottle| be the result of [=/obtaining a local storage bottle map=] given |environment| and "`web-locks`".
+1. If |bottle| is failure, then return failure.
+1. Return |bottle|'s associated [=/lock manager=].
 
 </div>
 
-<aside class=note>
-
-    There is an equivalence between the following:
-
-    * Agents that share a [=lock manager=].
-    * Agents that share a [=/storage shelf=], i.e. a per-origin [=localStorage|local storage area=] [[HTML]], set of [[IndexedDB-2#database-construct|databases]] [[IndexedDB-2]], or [[Service-Workers#cache-objects|caches]] [[Service-Workers]].
-
-</aside>
-
-<aside class=issue>
-Migrate this definition to [[HTML]] or [[Storage]] so it can be referenced by other standards.
```

Comment:
Perhaps another issue to incorporate LockManager to the bottle (or somewhere) properly?

Reply:
I think this would be done with proposed rewording above?

Reply:
Yes unless @annevk disagrees. We still have a yet another issue. for how to get the lock manager properly from the given environment, though.